locales/en/yourloops.json

```diff
@@ -237,6 +237,7 @@
   "member": "Member",
   "members": "Members",
   "menu-contact-support": "Customer support",
+  "message-from-current-settings-on-device": "This view shows the current settings, history of change and device information, but it looks like the system hasn't sent data yet.",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏I think a key explaining that's an alert message would be more appropriated, like `no-settings-on-device-alert-message` or something like that. Also you forgot to remove the other translation in `translation.json` file.